src/picongpu/include/plugins/radiation/radFormFactor.hpp

```diff
@@ -44,17 +44,18 @@ namespace picongpu
        */
       HDINLINE float_X operator()(const float_X N, const float_X omega, const vector_X observer_unit_vec) const
       {
+        //util::Pow<float_X, 1> pow;
+        const float_X sincX = util::pow( math::sinc( observer_unit_vec.x() * CELL_WIDTH / ( SPEED_OF_LIGHT * float_X( 2.0 ) ) * omega ), 1 );
+        const float_X sincY = util::pow( math::sinc( observer_unit_vec.y() * CELL_HEIGHT / ( SPEED_OF_LIGHT * float_X( 2.0 ) ) * omega ), 1 );
+        const float_X sincZ = util::pow( math::sinc( observer_unit_vec.z() * CELL_DEPTH / ( SPEED_OF_LIGHT * float_X( 2.0 ) ) * omega ), 1 );
           return math::sqrt(
-              N + ( N * N - N ) * util::square(
-                  math::sinc( observer_unit_vec.x() * CELL_WIDTH / ( SPEED_OF_LIGHT * float_X( 2.0 ) ) * omega ) *
-                  math::sinc( observer_unit_vec.y() * CELL_HEIGHT / ( SPEED_OF_LIGHT * float_X( 2.0 ) ) * omega ) *
-                  math::sinc( observer_unit_vec.z() * CELL_DEPTH / ( SPEED_OF_LIGHT * float_X( 2.0 ) ) * omega )
-              ) // util::square
+              N + ( N * N - N ) * util::square( sincX * sincY * sincZ )
```

Comment:
note: squares (^N) can be implemented more efficiently than the more general pow (^N)
note2: but with the `pow` implementation right now you are right